docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

```diff
@@ -774,6 +774,37 @@ suite:
 
 ---
 
+### `smartRetry`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Specifies the retry strategy to apply for that suite. It should be used along with [retries](#retries).
```

Comment:
"should" in this context would mean that it's a strong recommendation. But it's not. It won't do anything _unless_ this feature is used with retries.

```suggestion
Specifies the retry strategy to apply for that suite. Requires [retries](#retries) to be >= 1.
```